libs/blocks/table/table.js

```diff
@@ -5,7 +5,7 @@ import { decorateButtons } from '../../utils/decorate.js';
 const DESKTOP_SIZE = 900;
 const MOBILE_SIZE = 768;
 const tableHighlightLoadedEvent = new Event('milo:table:highlight:loaded');
-
+let tableIndex = 0;
```

Comment:
Since this is a global variable this would cause issues if there are more that one tables on the page. Could you verify and fix?

Reply:
Yes that is the point.
A unique ID must be assigned to the header; So, if we have multiple tables, id should start with table index, for example, **t1-c1-header, t1-c2-header, t2-c1-header**
We've put the entire cell as a column header, instead we've changed just the title to a column header, and given `aria-describedby` attr to describe the description and price in a11y.

<img width="964" alt="Screenshot 2024-08-01 at 12 49 21 PM" src="https://github.com/user-attachments/assets/b0db3b58-64be-49de-87ee-d301c09d3806">